### PR TITLE
Add a hedera-mirror-web3 Helm chart

### LINKED ISSUE #2956: Web3 Helm chart
### Problem

We need a way to deploy the new web3 API and continuously improve upon it.

### Solution

- Add a `hedera-mirror-web3` helm chart similar to gRPC helm chart
- Add a Grafana dashboard to show web3 graphs
- Add log and metric alerts
- Bump all chart dependencies

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
* Add a hedera-mirror-web3 Helm chart
* Add a slow JPA query alert to Java modules
* Add option for non-resource metrics to `HorizontalPodAutoscaler`
* Add option for scale up/down behavior to `HorizontalPodAutoscaler`
* Add web3 API to `docker-compose.yml`
* Bump chart dependencies to latest
* Change `PodDisruptionBudget` from the [deprecated](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125) `policy/v1beta1` to `policy/v1`
* Change Rosetta chart to use Postman/Newman for Helm tests
* Fix replacer not updating package-lock.json v2 properly
* Rename `Log4j2Errors` alert to the more generic `LogErrors`

**Related issue(s)**:

Fixes #2956 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-mirror-web3/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+icon: https://camo.githubusercontent.com/cca6b767847bb8ca5c7059481ba13a5fc81c5938/68747470733a2f2f7777772e6865646572612e636f6d2f6c6f676f2d6361706974616c2d686261722d776f72646d61726b2e6a7067
+maintainers:
+  - name: ijungmann
+  - name: Nana-EC
+  - name: steven-sheehy
+  - name: xin-hedera
+name: hedera-mirror-web3
+sources:
+  - https://github.com/hashgraph/hedera-mirror-node
+version: 0.34.0-SNAPSHOT
```
Comment (xin-hedera) [inline charts/hedera-mirror-web3/Chart.yaml:14]:
```suggestion
version: 0.35.0-SNAPSHOT
```

**Review by xin-hedera — DISMISSED** [review]
LGTM. just need to update the charts snapshot version

File: charts/hedera-mirror-web3/Chart.yaml (hunk 1 above)
Comment (steven-sheehy) [reply charts/hedera-mirror-web3/Chart.yaml]: Fixed that and updated replacer for web3. Also added web3 to docker-compose

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**